src/java.management/share/classes/com/sun/jmx/remote/security/MBeanServerAccessController.java

```diff
@@ -74,13 +72,8 @@
  * be overridden, for instance if the default checking behavior is
  * inappropriate.</p>
  *
- * <p>If there is no SecurityManager, then the access controller will refuse
- * to create an MBean that is a ClassLoader.  This prevents
- * people from opening security holes unintentionally. Otherwise, it
- * would not be obvious that granting write access grants the ability to
- * download and execute arbitrary code in the target MBean server. Advanced
- * users who do want an MBean which is a ClassLoader are presumably advanced enough
- * to handle policy files and security managers.</p>
+ * <p>An MBean could possibly be a ClassLoader.  Users who do not want an
+ * MBean which could be a ClassLoader should not permit MBean creation.</p>
```

Comment:
Maybe we should just refuse to create an MBean which is a ClassLoader for now (see further comments below) and handle that part in a dedicated PR.

Reply:
Yes agreed I'm going to update again with checkClassLoader put back.